static/app/components/clippedBox.tsx

```diff
@@ -16,7 +17,18 @@ type DefaultProps = {
 type Props = {
   clipHeight: number;
   className?: string;
+  /**
+   * When available replaces the default clipFade component
+   */
+  clipFade?: ({showMoreButton}: {showMoreButton: React.ReactNode}) => React.ReactNode;
```

Comment:
What about having this accept a component?

Reply:
yes maybe it would be better, but this component needs to be refactored so I think I will do it in a separate PR